### PR TITLE
bring mp ls into tycho build structure

### PR DESCRIPTION
**Pull request by ajm01**
Signed-off-by: Andrew J. Mauer <ajmauer@gmail.com>

### CODE REVIEW

File: releng/target-platform/target-platform.target [hunk 1]
```diff
@@ -53,16 +53,35 @@
...
+		</location>		
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="https://download.eclipse.org/jdtls/snapshots/repository/1.5.0.202109201721/"/>
+			<unit id="org.eclipse.jdt.ls.core" version="1.5.0.202109201721"/>
+		</location>
+		<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+			<repository location="http://download.eclipse.org/lsp4mp/releases/0.4.0/repository/"/>
+			<unit id="org.eclipse.lsp4mp.jdt.core" version="0.4.0.20220324-1655"/>
+		</location>
+		<location includeDependencyDepth="none" includeDependencyScopes="compile" includeSource="true" missingManifest="generate" type="Maven">
```
Comment (scottkurz) [inline releng/target-platform/target-platform.target:76]: get rid of

File: features/io.openliberty.wdt2.feature/feature.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <feature
       id="io.openliberty.wdt2.feature"
-      label="Feature"
+      label="WDT2"
```
Comment (scottkurz) [inline features/io.openliberty.wdt2.feature/feature.xml:4]: WDT => LDT

File: releng/target-platform/target-platform.target (hunk 1 above)
Comment (ajm01) [reply releng/target-platform/target-platform.target:76]: git rid of line 76 you mean?

**Review by scottkurz — COMMENTED** [review]
I couldn't seem to install this on a 2021-12 base.   Leaving these comments then will try the latest 22-03.

**Comment by scottkurz** [comment]
So along with the comments I already made, the other TODOs I see are:

- add copyright attribution to projects we copied from (after listing the projects and confirming the licenses, e.g. ASL 2, etc.)
- add mp jdt plugin to feature install
- For snippet code gen via MP LS, fix template variables getting written into Java w/o substitution
- move to a  2202-03 level for plugins across the board.   Let's just try to move everything there instead of possibly continuing to have incompatibilities.  
- confirm with feature install we can use LS function in Java editor, not just generic text editor

Except for the copyright attribute thing we can do the rest in any number of follow-up issues, 1-N, however.       Can you make sure we have GH issue(s) for whatever's not immediately on your plate and will need to be followed up on?

**Pull request merged**